### DISCUSSION TITLE
WIP - Use case: Different types of episodes on the same feed

### DISCUSSION THREAD

**Discussion by dellagustin** [Podcastindex-org/podcast-namespace#486] [General]
This is not exactly a proposal, but a description for use case(s) related to having different types of episodes on the same feed.

Some times, these types of episodes are so different that they could be considered different podcasts, other times they are like "sub-series" of a same podcast.

Namespace support for this sue case could enable features of filtering for this "sub-series" (in or out).

**Comment by theDanielJLewis** [comment]
Maybe replicating and then expanding the `<itunes:type>` tag? That currently supports “full” (default), “bonus,” and “trailer.” The latter two are intended to be attached to something. So a “bonus episode” would be a bonus _to_ another episode, season, or the whole show. But I think another type to indicate a special standalone episode/miniseries would be good. It could be “miniseries,” “special,” “feeddrop,” “promo,” or more.

**Reply by keunes** [reply]
I was looking into the `<itunes:type>` tag and landed here. But seems that you meant to refer to the `<itunes:episodeType>` tag ([apple docs](https://help.apple.com/itc/podcasts_connect/#/itcb54353390)).
